File: Examples/PHP/Folders/DeleteFolder.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Storage\StorageApi;
use Aspose\Storage\AsposeApp;

class Folder {

	public $storageApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->storageApi = new StorageApi();
	}

	public function deleteFolder() {
		// First create folder
		$result = $this->storageApi->PutCreateFolder($Path="testfolder", $storage = null, $destStorage = null);

		// Now delete folder
        $result = $this->storageApi->DeleteFolder($Path="testfolder", $storage = null, $recursive = null);
        print_r($result);
	}
}

$folder = new Folder();
$folder->deleteFolder();

?>
//ExEnd:

File: Examples/Ruby/Folders/delete_folder.rb
#ExStart:
require 'aspose_storage_cloud'

class Folder
    
  include AsposeStorageCloud

  def initialize
    # Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("App Key", "App SID")
    @storage_api = StorageApi.new
  end

  def create_folder(folder_name)
  	response = @storage_api.put_create_folder(folder_name)
  end

  def delete_folder
    folder_name = "testfolder"

    # First create folder
	  create_folder(folder_name)

  	# Now delete this folder
	  response = @storage_api.delete_folder(folder_name)
  end

end

folder = Folder.new()
puts folder.delete_folder
#ExEnd: